### PR TITLE
Clarify secure communications and fix elb hub config

### PR DESCRIPTION
**Pull request by zkayyali812**

### CODE REVIEW

File: stories/topics/ref-aap-gcp-external-load-balancer.adoc [hunk 1]
```diff
@@ -26,32 +26,41 @@ If you do not see *Network Services*, then select *View All Products*.
 . Select the *HTTP(S) Load Balancing* tile and click btn:[START CONFIGURATION].
 . Select *From Internet to my VMs* or *Serverless services and Global HTTP(s) Load Balancer*.
 . Click btn:[CONTINUE].
+. Provide your load balancer a name, example - `<DeploymentName>-aap-<cntrlr/hub>-ext-lb`
```
Comment (ianf77) [inline stories/topics/ref-aap-gcp-external-load-balancer.adoc:29]: a name, for example, `<DeploymentName

File: stories/topics/ref-aap-gcp-external-load-balancer.adoc [hunk 2]
```diff
@@ -26,32 +26,41 @@ If you do not see *Network Services*, then select *View All Products*.
...
 . Ensure that *Frontend* configuration is selected at the left of the screen.
-. Enter the required fields on the *Frontend* configuration page.
+. Enter the required fields on the *Frontend* configuration page then click btn:[DONE]
```
Comment (ianf77) [inline stories/topics/ref-aap-gcp-external-load-balancer.adoc:31]: configuration page.
. Click btn:[DONE]

File: stories/topics/ref-aap-gcp-external-load-balancer.adoc [hunk 3]
```diff
@@ -26,32 +26,41 @@ If you do not see *Network Services*, then select *View All Products*.
...
 .. *Protocol*: HTTPS.
 .. *Port*: 443.
-.. *IP Address* Select an existing IP addres or create a new one.
+.. *IP Address* Select an existing IP address or create a new one.
 .. *Certificate*, You can use your own SSL Certificate or use a Google managed certificate.
 .. *SSL Policy*: GCP default, or another configured policy.
 . Click btn:[Backend configuration] from the menu on the left.
 . Click the *Backend services & backend buckets* drop-down.
 . Click btn:[CREATE A BACKEND SERVICE].
+.. Provide your backend service a name, example - `<DeploymentName>-aap-<cntrlr/hub>-ext-lb-bknd-svc`
```
Comment (ianf77) [inline stories/topics/ref-aap-gcp-external-load-balancer.adoc:40]: , for example,

File: stories/topics/ref-aap-gcp-external-load-balancer.adoc (hunk 1 above)
Comment (zkayyali812) [reply stories/topics/ref-aap-gcp-external-load-balancer.adoc]: Done

File: stories/topics/ref-aap-gcp-external-load-balancer.adoc (hunk 2 above)
Comment (zkayyali812) [reply stories/topics/ref-aap-gcp-external-load-balancer.adoc]: Done

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**